google-youtube-upload.html

```diff
@@ -393,6 +422,30 @@
         value: Boolean('MediaRecorder' in window &&
                        'mediaDevices' in navigator &&
                        'getUserMedia' in navigator.mediaDevices)
+      },
+
+      /**
+       * The [MediaStream audio constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/audio).
+       */
+      audio: {
+        type: Object,
+        value: true
+      },
+
+      /**
+       * The [MediaStream video constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/video).
+       */
+      video: {
+        type: Object,
+        value: true
+      },
+
+      /**
+       * The media type.
+       */
+      mimeType: {
+        type: String,
+        value: ''
```

Comment:
@e111077 What if the browser doesn't support WebM container and/or VP8/VP9 codecs? The current W3C specification states that if the MIME type is not specified, the browser will use a platform-specific default format: https://w3c.github.io/mediacapture-record/#dom-mediarecorderoptions-mimetype.